### PR TITLE
Use v2Availability, add messaging

### PR DESCRIPTION
**Pull request by cvalarida**
## Record not found error
image

## Everything else
image

### CODE REVIEW

File: src/js/claims-status/containers/AppealInfo.jsx [hunk 1]
```diff
@@ -82,17 +103,17 @@ export class AppealInfo extends React.Component {
...
+      appealContent = appealsDownMessage;
     } else if (appealsAvailability === BACKEND_SERVICE_ERROR) {
-      appealContent = systemDownMessage;
+      appealContent = appealsDownMessage;
     } else if (appealsAvailability === FETCH_APPEALS_ERROR) {
-      appealContent = systemDownMessage;
+      appealContent = appealsDownMessage;
     } else {
-      appealContent = systemDownMessage;
+      appealContent = appealsDownMessage;
```
Comment (cvalarida) [inline src/js/claims-status/containers/AppealInfo.jsx]: @gnakm Verifying: We don't actually want the generic system down message to appear at all, is that correct? We want to use this new messaging whenever something goes wrong?

Comment (gnakm) [reply src/js/claims-status/containers/AppealInfo.jsx]: Yes!

Comment (gnakm) [reply src/js/claims-status/containers/AppealInfo.jsx]: Yass please.

**Review by gnakm — APPROVED** [review]
lgtm! 🎉

File: src/js/claims-status/containers/AppealInfo.jsx [hunk 2]
```diff
@@ -25,6 +24,28 @@ import {
...
+    </div>
+  </div>
+);
+
+const recordsNotFoundMessage = (
+  <div className="row" id="recordsNotFoundMessage">
+    <div className="small-12 columns">
+      <div className="react-container">
+        <h3>We’re sorry. We can’t find your records in our system.</h3>
+        <p>If you think they should be here, please try again later or call the Vets.gov Help Desk at <a href="tel:18555747286">1-855-574-7286</a> (TTY: <a href="tel:18008294833">1-800-829-4833</a>). We’re here Monday–Friday, 8:00 a.m.–8:00 p.m. (ET).</p>
```
Comment (cehsu) [inline src/js/claims-status/containers/AppealInfo.jsx]: It looks like this design has been updated from the guidance provided in the linked issue. @gnakm has signed off on the updated content, so it must be good to go.

File: src/js/claims-status/containers/AppealInfo.jsx [hunk 3]
```diff
@@ -25,6 +24,28 @@ import {
   AVAILABLE,
 } from '../utils/appeals-v2-helpers';
 
+const appealsDownMessage = (
+  <div className="row" id="appealsDownMessage">
+    <div className="small-12 columns">
+      <div className="react-container">
+        <h3>We’re sorry. Something went wrong on our end.</h3>
+        <p>Please refresh this page or try again later. If it still doesn’t work, you can call the Vets.gov Help Desk at <a href="tel:18555747286">1-855-574-7286</a> (TTY: <a href="tel:18008294833">1-800-829-4833</a>). We’re here Monday–Friday, 8:00 a.m.–8:00 p.m. (ET).</p>
```
Comment (cehsu) [inline src/js/claims-status/containers/AppealInfo.jsx]: We typically add the + to the href for telephone numbers: href="tel:+1-800-827-1000", we may consider doing so here.

File: src/js/claims-status/containers/AppealInfo.jsx [hunk 4]
```diff
@@ -25,6 +24,28 @@ import {
...
+      </div>
+    </div>
+  </div>
+);
+
+const recordsNotFoundMessage = (
+  <div className="row" id="recordsNotFoundMessage">
+    <div className="small-12 columns">
+      <div className="react-container">
```
Comment (cehsu) [inline src/js/claims-status/containers/AppealInfo.jsx]: Should we add renaming this class to a style cleanup epic?

File: src/js/claims-status/containers/AppealInfo.jsx [hunk 5]
```diff
@@ -82,17 +103,17 @@ export class AppealInfo extends React.Component {
       // Yes, we have your appeals. No, the one you requested isn't one of them.
       appealContent = <AppealNotFound/>;
     } else if (appealsAvailability === USER_FORBIDDEN_ERROR) {
```
Comment (cehsu) [inline src/js/claims-status/containers/AppealInfo.jsx]: It looks like there are only two cases here, can we consolidate these into a smaller block of code?

File: test/claims-status/components/appeals-v2/AppealInfo.unit.spec.jsx [hunk 6]
```diff
@@ -76,39 +76,37 @@ describe('<AppealInfo/>', () => {
...
   });
 
   it('should render system down message when FETCH_APPEALS_ERROR present', () => {
     const props = { ...defaultProps, appealsAvailability: FETCH_APPEALS_ERROR };
     const wrapper = shallow(<AppealInfo {...props}/>);
-    expect(wrapper.find('#systemDownMessage').length).to.equal(1);
+    expect(wrapper.find('#appealsDownMessage').length).to.equal(1);
   });
 
   it('should render system down message when other error present', () => {
```
Comment (cehsu) [inline test/claims-status/components/appeals-v2/AppealInfo.unit.spec.jsx]: It looks like we might want to update the descriptions of the formerly "system down" tests.

**Review by cehsu — APPROVED** [review]
LGTM. Had a few minor questions and comments.

File: src/js/claims-status/containers/AppealInfo.jsx (hunk 4 above)
Comment (cvalarida) [reply src/js/claims-status/containers/AppealInfo.jsx]: Yeah, that sounds like a great idea. Do we have a style cleanup epic?

File: src/js/claims-status/containers/AppealInfo.jsx (hunk 2 above)
Comment (cvalarida) [reply src/js/claims-status/containers/AppealInfo.jsx]: I must be missing something. What's the difference?

File: test/claims-status/components/appeals-v2/AppealInfo.unit.spec.jsx (hunk 6 above)
Comment (cvalarida) [reply test/claims-status/components/appeals-v2/AppealInfo.unit.spec.jsx]: Good eye as usual! :female_detective:

File: src/js/claims-status/containers/AppealInfo.jsx (hunk 5 above)
Comment (cvalarida) [reply src/js/claims-status/containers/AppealInfo.jsx]: Yeah, I thought about consolidating them all. I guess I'm more assuming we'll have a reason to break out the availability into the different "types" of unavailable in the first place, so this leaves it more open to modification.

On the other hand, if it needs to change later, it's super easy to pull out a specific case, so... :man_shrugging: 

I'll go ahead and consolidate these. It's cleaner. :+1:

**Pull request merged**